CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 # Changelog
 
-# 1
+# 2
+
+## 2.0.0
+
+WARNING: If you upgrade to this version, any new writes to blind credentials
+will be in a format that is only compatible in 1.11.0 forward. If you've
+upgraded and need to downgrade, you should downgrade to 1.11.0.
```

Comment:
Maybe add explicit instructions for how to upgrade? e.g. "If your blind credentials table is not empty, follow these steps"

Reply:
Yeah, I'll add some more info here and will add an "Upgrading" section to the docs.

Reply:
Done